pathway/templates/_helpers/multisave.html

```diff
@@ -1 +1,16 @@
-<div initialise-empty={{ initialise_empty }} save-multiple="{{ model }}" save-multiple-form-url="'{{ form_url }}'" save-multiple-label="'{{ label }}'"></div>
+{% load forms %}
+<div save-multiple-wrapper="{{ model }}" initialiseEmpty="{{ initialise_empty }}">
+  <div ng-repeat="editing in model.subrecords track by $index">
+    <div class="form-horizontal">
+      {% include subrecord.get_form_template %}
+    </div>
+    <div class="row content-offset-below-25">
+      <div class="col-sm-11">
+        <button ng-click="remove($index)" class="pull-right btn btn-secondary">{% icon "fa-trash" %}Remove</button>
+      </div>
```

Comment:
What happens if I double click these buttons ?
(Is this a good example of why we should have a button tag?)